src/Events/payments.js

```diff
@@ -12,7 +12,8 @@ const cardIcons = {
   'default': require('../../assets/icon-visa-pay.png'),
 }
 
-const URI =  'http://localhost:3000' // 'https://staging.bigneon.com'
+const URI = 'https://staging.bigneon.com'
```

Comment:
I changed the url to
```suggestion
const tmp_username=''
const tmp_password=''
const URI = `https://${tmp_username}:${tmp_password}@staging.bigneon.com`
```
![screenshot 2018-11-28 at 08 04 38](https://user-images.githubusercontent.com/41575888/49132338-b781c300-f2e4-11e8-96ee-4773ab15c33d.png)

Reply:
Maybe this needs to come out into a constant:

[src/constants/Server.js#L3](https://github.com/revelrylabs/bn-mobile-react/blob/6749435b4050793629783ec629b89a3804862304/src/constants/Server.js#L3)
`export const BASE_URL = 'https://username:password@staging.bigneon.com`

Then

[src/constants/Server.js#L4](https://github.com/revelrylabs/bn-mobile-react/blob/6749435b4050793629783ec629b89a3804862304/src/constants/Server.js#L4)
```export const server = new Bigneon.Server({prefix: `${BASE_URL}/api`})//, {}, mocker)```

Then the above file can import `BASE_URL` so we don't need to keep changing it?